assets/src/scss/layout/navbar/languages.scss

```diff
@@ -32,7 +32,6 @@
 }
 
 .nav-languages {
-  display: none;
```

Comment:
Not necessary to set this attributes. The layout should be defined as default and any `nav` (Desktop and Mobile) would be in charge to `show` or `hide`